front/public/locales/en/timesStops.json

```
@@ -2,6 +2,7 @@
   "arrivalTime": "Requested arrival Time",
   "calculatedArrivalTime": "Calculated arrival time",
   "calculatedDepartureTime": "Calculated departure time",
+  "d+": "D+",
```

The key is not very explicit. Maybe we could do that ? How about passing the count inside also ?

```suggestion
  "dayCounter": "D+",
```

---

done

---

Thx. And for the `count` ? You think it's not needed in the trad if we don't handle plurals ?

---

done